internal/config/config.go

```diff
@@ -661,6 +659,27 @@ func bgpAdvertisementsFromLegacyCR(ads []metallbv1beta1.LegacyBgpAdvertisement,
 	return ret, nil
 }
 
+func (ad *BGPAdvertisement) communtiesForBGPAd(adCommunities []string, communities map[string]uint32) error {
```

Comment:
typo: communties
Also, communitiesFor... implies that this is returning an object, which is not.
Does this have to be a method? It would be clearer as a function and having the community map assigned outside.

Reply:
fixed